cli/src/tests.rs

```diff
@@ -973,6 +973,73 @@ async fn setup_shamir(tmp_path: TmpPath) {
     // TODO list shamir setup
 }
 
+#[rstest::rstest]
+#[tokio::test]
+async fn workspace_import_file(tmp_path: TmpPath) {
+    let _ = env_logger::builder().is_test(true).try_init();
+    let tmp_path_str = tmp_path.to_str().unwrap();
+    let config = get_testenv_config();
+    let (url, [alice, ..], _) = run_local_organization(&tmp_path, None, config)
+        .await
+        .unwrap();
+
+    set_env(&tmp_path_str, &url);
+
+    let client = libparsec::internal::Client::start(
+        Arc::new(
+            ClientConfig {
+                with_monitors: false,
+                ..Default::default()
+            }
+            .into(),
+        ),
+        libparsec::internal::EventBus::default(),
+        alice.clone(),
+    )
+    .await
+    .unwrap();
+
+    // Create the workspace used to copy the file to
+    let wid = client
+        .create_workspace("new-workspace".parse().unwrap())
+        .await
+        .unwrap();
+    client.ensure_workspaces_bootstrapped().await.unwrap();
+
+    // Create a file to copy
+    let file = tmp_path.join("test.txt");
+    std::fs::write(&file, "Hello, World!").unwrap();
```

Comment:
It would be better to have a file that is bigger than the buffer size internally used (so > 4096), and not aligned on it, so typically a 10ko file:
```
let buff = "1234567890".repeat(1000);
assert_eq!(buff.len(), 10000);
```